evalme/text/text.py

```diff
@@ -58,9 +58,14 @@ def intersection(self, item, label_weights=None, algorithm=None, qval=None, per_
                 prefix = pred_value.get('when_label_value', '')
                 if prefix:
                     prefix += ':'
-                for l in pred_value[self._shape_key]:
-                    total_score[prefix + l] += best_matching_score
-                    total_weight[prefix + l] += 1
+                try:
+                    for l in pred_value[self._shape_key]:
+                        total_score[prefix + l] += best_matching_score
+                        total_weight[prefix + l] += 1
+                except KeyError:
+                    for l in pred_value[self.SHAPE_KEY]:
```

Comment:
1. check `self._shape_key in pred_value` before the loop
2. logger.error()

Reply:
1. Fixed
2. It's not an error, we have 2 types of html labels until everybody migrate to new labels.